base/templates/base.html

```diff
@@ -18,8 +18,7 @@
                     </form>
                 </div>
             {% else %}
-                <!-- TODO broken for some reason. should use url with reverse -->
-                <a class="navbar-brand" href="/">MySpotifyTool</a>
+                <a class="navbar-brand" href="{% url 'base:index' %}">MySpotifyTool</a>
```

Comment:
I don't think I actually fixed it. But I at least added a test case to show if it's happening. We should get you to checkout this branch and see if it's busted. If it is, I have a few things we can try. If it isn't, then we can just not merge this in.